runtime/drivers/olap.go

```diff
@@ -741,6 +744,15 @@ func (d Dialect) GetTimeExpr(t time.Time) (bool, string) {
 	}
 }
 
+func (d Dialect) DictGetExpr(lookupTable, lookupColumn, lookupKey string) string {
+	switch d {
+	case DialectClickHouse:
+		return fmt.Sprintf("dictGet('%s', '%s', %s)", lookupTable, lookupColumn, lookupKey)
+	default:
```

Comment:
Can you maybe add a comment about why it's not handling Druid or DuckDB? Like you wrote in the other PR comment:
> Druid already does reverse lookup inherently so defining lookup expression directly in dimension expression should be ok. For Duckdb I think we should just avoid going into this complexity as it should not matter much at that scale.